2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -1400,5 +1400,95 @@ Diversi ruoli risultano avere un monte ore minore rispetto a quello preventivato
   caption: "Monitoraggio ore settimo sprint"
 )
 
+=== Burndown-chart
+[da inserire]
+
+
+== Ottavo Sprint dal 26-12-2023 al 31-12-2023
+
+=== Obiettivi raggiunti
+- Miglioramento del Glossario con gestione di plurali e acronimi dei termini ed evidenziazione _case insensitive_;
+- Implementato il tracciamento requisito-fonte nella Analisi dei Requisiti;
+- Aggiornamento del documento Norme di Progetto;
+- Realizzato sito web per la documentazione;
+- Automatizzata l'individuazione dei termini del glossario nei documenti.
+- Realizzazione di una dashboard preliminare Grafana;
+- Unione dei diversi esploratori tecnologici in un singolo PoC.
+
+=== Obiettivi mancati
+- La dashboard di monitoraggio non implementa tutte le metriche individuate nel Piano di Qualifica;
+- Rimane da comprendere a fondo le implicazioni della scelta tra Next.js ed Express.js;
+- Permangono bug nelle GitHub Actions.
+
+=== Analisi problematiche insorte
+Durante il meeting di retrospettiva sono insorte le seguenti problematiche:
+
+- i lavori hanno subito rallentamenti dovuti al periodo di festività.
+
+=== Risoluzioni attuate
+Nessuna.
+
+=== Analisi costi di sprint effettivi
+
+#set text(
+  size: 10pt
+)
+
+#figure(
```

Comment:
Mancano da compilare i campi "Totale ore ruolo" e "Totale"
Manca il codice per evidenziare l'intestazione della tabella:
`fill: (col, row) => if col == 0 and row > 8 or row == 0 {rgb("#bbbbbb")},`